### PR TITLE
Package vue-jsoo.0.2

### PR DESCRIPTION
**Pull request by ocpmax**
### `vue-jsoo.0.2`
Binding of Vue_js

---
* Homepage: https://gitlab.com/o-labs/vue-jsoo
* Source repo: git+https://gitlab.com/o-labs/vue-jsoo.git
* Bug tracker: https://gitlab.com/o-labs/vue-jsoo/-/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

**Comment by ocpmax** [comment]
There seem to be an issue with the osx image that crashes during a brew update

File: packages/vue-jsoo/vue-jsoo.0.2/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+synopsis: "Binding of Vue_js"
+maintainer: ["maxime.levillain@origin-labs.com"]
+authors: ["Maxime Levillain"]
+license: "MIT"
+homepage: "https://gitlab.com/o-labs/vue-jsoo"
+doc: "https://o-labs.gitlab.io/vue-jsoo/vue-jsoo"
+bug-reports: "https://gitlab.com/o-labs/vue-jsoo/-/issues"
+depends: [
```
Comment (mseri) [inline packages/vue-jsoo/vue-jsoo.0.2/opam:9]:
```suggestion
depends: [
  "ocaml" {>= "4.05"}
```

Some list functions that are used in the project were introduced only in more recent versions of the compiler

**Comment by mseri** [comment]
> There seem to be an issue with the osx image that crashes during a brew update (in other pull requests too)

Yes, there are sometimes these brew issues on mac. It is not s problem for this PR. I think the only change needed was a lower bound on the supported ocaml version

**Comment by mseri** [comment]
Looks good. Feel free to announce it on the discuss.ocaml.org forum. We have a community category and an announcement tag for this purpose

**Pull request merged**